ouroboros-consensus-cardano/src/shelley/Ouroboros/Consensus/Shelley/Eras.hs

```diff
@@ -161,6 +163,18 @@ class ( Core.EraSegWits era
          , SL.Validated (Core.Tx era)
          )
 
+  -- | Get the protocol version out of a 'Core.PParamsUpdate', used to detect
+  -- whether we should perform a HF. This will likely be removed/changed once we
+  -- implement HF enactment in Conway (see
+  -- <https://github.com/input-output-hk/ouroboros-consensus/issues/61>).
+  --
+  -- For now, this always returns 'Nothing' for Conway (see the instance below).
+  getProposedProtocolVersion :: Core.PParamsUpdate era -> Maybe ProtVer
+  default getProposedProtocolVersion ::
```

Comment:
I wasn't aware of that, we have lots of existing default method impls: `rg -t hs '  default '` returns 26 matches. Might make sense to add it to the style guide

In any case, this should be purely temporary until #61; I just went this route as it resulted in the smallest diff, but I agree that it is usually best to avoid default method impls.

Reply:
I added a commit which removes the default implementation - but I can remove it  if it seems not worth it (what with the fix being temporary).